roxie/ccd/ccdserver.cpp

```diff
@@ -1376,6 +1377,20 @@ class CRoxieServerActivity : implements CInterfaceOf<IRoxieServerActivity>, impl
             }
         }
 #endif
+        if (factory)
+        {
+            ActResetLog actVal = factory->queryQueryFactory().getActResetLogged();
+            if (actVal == ActResetSome)
```

Comment:
Not really correct, this could get called while another query of this type was running.